codebuild/spec/buildspec_fuzz_batch.yml

```diff
@@ -0,0 +1,53 @@
+---
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+# this file except in compliance with the License. A copy of the License is
+# located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing permissions and
+# limitations under the License.
+version: 0.2
+
+# Doc for batch https://docs.aws.amazon.com/codebuild/latest/userguide/batch-build-buildspec.html#build-spec.batch.build-list
+batch:
+  build-list:
+
+    - identifier: s2nFuzzerOpenSSL111Coverage
+      buildspec: codebuild/spec/buildspec_ubuntu_fuzz_artifacts.yml
+      env:
+        privileged-mode: true
+        compute-type: BUILD_GENERAL1_SMALL
```

Comment:
how did you decide to use SMALL here?

Reply:
Looks like it was adopted from here: https://docs.aws.amazon.com/codebuild/latest/userguide/build-env-ref-compute-types.html

Reply:
Good catch- it should match the current instance size from [here](https://github.com/awslabs/s2n/blob/main/codebuild/codebuild.config#L12)

Reply:
I mean, why small, and not medium or large, etc

Reply:
Copy/paste error?  For fuzzing we do want more cores, a Large or 2XL would be best.